src/components/chart/chart.tsx

```diff
@@ -116,6 +117,12 @@ export class Chart {
         totalRange: number;
     };
 
+    /**
+     * Fired when a feed item with `clickable` set to `true` is clicked
+     */
+    @Event()
+    public itemSelected: EventEmitter<ChartItem>;
```

Comment:
"selected" implies that we are keeping track of selection state. Are we?

If not, I think we should call this something else. We use `interact` in several components already.

Reply:
Good point! I'll fixup!